### PR TITLE
client/web: add metric for exit node advertising

### PR DESCRIPTION
**Pull request by tendstofortytwo**
Updates https://github.com/tailscale/corp/issues/15215.

### CODE REVIEW

File: client/web/web.go [hunk 1]
```diff
@@ -473,6 +476,22 @@ func (s *Server) servePostNodeUpdate(w http.ResponseWriter, r *http.Request) {
...
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+
+	isCurrentlyExitNode := slices.Contains(prefs.AdvertiseRoutes, exitNodeRouteV4) || slices.Contains(prefs.AdvertiseRoutes, exitNodeRouteV6)
+
+	if postData.AdvertiseExitNode != isCurrentlyExitNode {
+		if postData.AdvertiseExitNode {
+			s.lc.IncrementCounter(r.Context(), "web_client_advertise_exitnode_enable", 1)
```
Comment (willnorris) [inline client/web/web.go:489]: could you go ahead and update the other existing `IncrementCounter` call at line 197 to use `r.Context()` ?

**Review by willnorris — APPROVED** [review]

File: client/web/web.go (hunk 1 above)
Comment (tendstofortytwo) [reply client/web/web.go:489]: done

File: client/web/web.go [hunk 2]
```diff
@@ -66,6 +66,11 @@ const (
 	sessionCookieExpiry = time.Hour * 24 * 30 // 30 days
 )
 
+var (
+	exitNodeRouteV4 = netip.MustParsePrefix("0.0.0.0/0")
+	exitNodeRouteV6 = netip.MustParsePrefix("::/0")
+)
```
Comment (soniaappasamy) [inline client/web/web.go:72]: I wonder if we actually have these vars somewhere in a lib... But this is a nice update!

**Review by soniaappasamy — APPROVED** [review]

File: client/web/web.go (hunk 2 above)
Comment (willnorris) [reply client/web/web.go:72]: we seem to have them at least 3 times in this repo.  maybe the copy in `net/netutil` would be the best to export and reuse?

Comment (willnorris) [reply client/web/web.go:72]: maybe in a separate PR though, since @bradfitz might have opinions on exporting those long-term?

**Pull request merged**